core-bundle/src/Resources/contao/library/Contao/DcaExtractor.php

```diff
@@ -421,7 +423,7 @@ protected function createExtract()
 						$table = substr($config['foreignKey'], 0, strrpos($config['foreignKey'], '.'));
 					}
 
-					$arrRelations[$field] = array_merge(array('table'=>$table, 'field'=>'id'), $config['relation']);
+					$arrRelations[$field] = array_merge(array('table'=>$table, 'field'=>'id', 'load'=>'lazy', 'type'=>null), $config['relation']);
```

Comment:
This should be fixed where the values are read and not where they are written.

Reply:
Shouldn't we make sure the relation data is always consistent?

Reply:
No, because that is a behavior change and thus a possible BC break.

Reply:
Changed in acc9110172ba8f3aa53526b159221c94013b0bb7